web/src/components/Testimonial/Testimonial.tsx

```diff
@@ -39,8 +39,9 @@ export const Testimonial: React.FC<TestimonialProps> = (
 		: "";
 
 	const testimonialClasses = classnames(
+		"testimonial",
 		styles.testimonial,
-		{
+		variant !== "default" && {
```

Comment:
Personally I'd use the one-liner with the combined logic rather than nested logic